pandas/tseries/index.py

```diff
@@ -827,16 +828,21 @@ def _format_native_types(self, na_rep=u('NaT'),
     def to_datetime(self, dayfirst=False):
         return self.copy()
 
-    def astype(self, dtype):
+    def astype(self, dtype, copy=True):
         dtype = np.dtype(dtype)
```

Comment:
can you add a doc-string here (and add versionadded 0.18.2 for ``copy``). This might need to be a shared -doc string (and prob need to add this in ``Index.astype`` as well.